src/components/FilterBar/TextFilter.js

```diff
@@ -1,61 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import {
-  Button,
-  GU,
-  IconSearch,
-  Popover,
-  SearchInput,
-  useLayout,
-} from '@1hive/1hive-ui'
+import { GU, Popover, SearchInput, useLayout } from '@1hive/1hive-ui'
 
 const TextFilter = React.memo(
-  ({
-    textFilter,
-    updateTextFilter,
-    placeholder = '',
-    openerRef,
-    visible,
-    setVisible,
-    onClick,
```

Comment:
Since we are removing this props, we should also remove them from the declared `PropTypes` below